app/kubernetes/views/applications/create/createApplication.html

```diff
@@ -631,6 +631,13 @@
                     </div>
                     <!-- !has-override -->
                   </div>
+
+                  <div class="my-2 w-full">
+                    <p class="vertical-center text-muted text-xs" ng-if="ctrl.secrets.length === 0">
```

Comment:
`<portainer-tooltip message="'There are no secrets available in this namespace.'"></portainer-tooltip>` might work and help consistency

Reply:
it is not a tooltip